### DISCUSSION TITLE
pdf  the bottom of the table is not closed

### DISCUSSION THREAD

**Discussion by ollie-zhang** [jsvine/pdfplumber#723] [Ask for help with specific PDFs]
<img width="652" alt="image" src="https://user-images.githubusercontent.com/56536125/188391484-8bc28f48-c92e-4794-b06b-108ca23409a6.png">

And following is my code,
pdf = pdfplumber.open("test.pdf")
p = pdf.pages[0]

extracted = p.extract_table()
print(extracted)
the latest raw is ['6654', None, None, None, 'manufacturer', '生产厂家', 'VARCHAR2', '10', '', '必填', '是', 'Manufacturer', ''], the raw[1:3] are None, can you give me some suggestion

**Comment by jsvine** [comment]
Hi @ollie-zhang, and thanks for you interest in this library. This discussion may be helpful for you: https://github.com/jsvine/pdfplumber/discussions/625

The general approach is to identify the vertical position of where the bottom line *should* be, and then passing that  position to the `explicit_horizontal_lines` table-extraction setting.
